### PR TITLE
create new package and create inside the class

### PR DESCRIPTION
**Pull request by bluee03**
please review

### CODE REVIEW

File: src/test/java/gitPracticeObjectRepository/CreateProductPage.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package gitPracticeObjectRepository;
+
+public class CreateProductPage {
```
Comment (123Rajesh123) [inline src/test/java/gitPracticeObjectRepository/CreateProductPage.java:3]: change the xpath. 
Do not hard code the date.

**Review by 123Rajesh123 — COMMENTED** [review]
added few comments

**Comment by bluee03** [comment]
fix

**Comment by 123Rajesh123** [comment]
mrge

**Pull request merged**